.github/workflows/ohos.yml

```diff
@@ -118,3 +118,88 @@ jobs:
+
+  build_harmonyos_aarch64:
+    name: HarmonyOS Build (aarch64)
+    continue-on-error: true
+    runs-on: hos-builder
+    # Todo: Change to servo/servo before merging and after getting token!
+    if: github.repository == 'jschwe/servo'
+    env:
+      CARGO_INCREMENTAL: 1
+    steps:
+      - if: ${{ github.event_name != 'pull_request_target' }}
+        run: git fetch --depth=1 origin $GITHUB_SHA
+      - if: ${{ github.event_name == 'pull_request_target' }}
+        run: git fetch --depth=1 origin ${{ github.event.pull_request.head.sha }}
+      - run: |
+          git switch --detach
+          git reset --hard FETCH_HEAD
+      - name: Build for aarch64 HarmonyOS
+        run: |
+          ./mach build --locked --target aarch64-unknown-linux-ohos --profile=${{ inputs.profile }} --flavor=harmonyos --no-default-features --features tracing,tracing-hitrace
+      - uses: actions/upload-artifact@v4
+        with:
+          # Upload the **unsigned** artifact - We don't have the signing materials in pull request workflows
+          path: target/openharmony/aarch64-unknown-linux-ohos/${{ inputs.profile }}/entry/build/harmonyos/outputs/default/servoshell-default-unsigned.hap
+          name: servoshell-hos-${{ inputs.profile }}.hap
+
+
+  test_harmonyos_aarch64:
+    name: Test HarmonyOS aarch64
+    # Don't block servos Merge queue on this job failing.
+    # Since we just added this, there might be some hidden issues,
+    # so in the beginning we will just do a best effort approach but ignore errors.
+    continue-on-error: true
+    runs-on: hos-runner
+    if: github.repository == 'jschwe/servo'
+    needs: build_harmonyos_aarch64
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          # Name of the artifact to download.
+          # If unspecified, all artifacts for the run are downloaded.
+          name: servoshell-hos-${{ inputs.profile }}.hap
+      - name: Test hdc device
+        # First we ensure a device is actually connected and working.
+        run: hdc list targets && hdc shell echo hello world
+      - name: Sign the hap
+        run: |
+          ls -la
+          /usr/bin/sign-hos.sh servoshell-default-unsigned.hap servoshell-hos-signed.hap
+      - name: Install
+        run: |
+          # Uninstall first. hdc is not very reliable in terms of exiting with an error, so we uninstall first
+          # to make sure we don't use a previous version if installation failed for some reason.
+          hdc uninstall org.servo.servoshell
+          hdc install -r servoshell-hos-signed.hap
+      - name: Test loading servo.org
+        env:
+          TRACE_BUFFER_SZ_KB: "524288" # 512 MB
+        run:  |
+          mkdir test_output
+          hdc shell aa force-stop org.servo.servoshell
+          hdc shell hitrace -b "${TRACE_BUFFER_SZ_KB}" app graphic ohos freq idle memory --trace_begin
```

Comment:
Could we add a comment describing why this command is needed?

Reply:
Added comments, please check

Reply:
Thanks! I was actually wondering more about the arguments passed to the `hitrace` command. If I understand your comment correctly, the `graphic ohos freq idle memory` are there to capture traces that could *eventually* be used for analyzing performance in different areas, but not required to just ensure the `PageLoadEndedPrompt` is captured in the trace. Is this correct?

Reply:
Yes, correct. Only the `app` category is needed to save servos trace events.
I thought it might be nice to save some more information, since it makes the trace more interesting to look at if you do decide to download and inspect the trace.